addons/web/static/src/views/list/list_renderer.js

```diff
@@ -1165,7 +1165,8 @@ export class ListRenderer extends Component {
         switch (hotkey) {
             case "tab": {
                 const index = list.records.indexOf(record);
-                if (index === list.records.length - 1) {
+                const lastIndex = list.model.editable === 'top' && !record.resId ? 0 : list.records.length - 1;
+                if (index === lastIndex) {
```

Comment:
Why add editable to the model ? In this use case, you can use this.props.editable ?
If you really want to add it to the model, you should add it to the DynamicRecordList.

Reply:
Because I'm a dummy :upside_down_face: